src/components/login-form.js

```diff
@@ -134,7 +135,8 @@ export default function LoginForm({
   ];
 
   const hasBlockingError = hasLoginFailedTooOften || isUserAccountLocked;
-  const canLoginWithShibboleth = allowExternalLogin && clientConfig.samlAuth?.identityProviders.length;
+  const canLoginWithShibboleth = samlIdentityProviders.length
+    && !clientConfig.disabledFeatures.includes(FEATURE_TOGGLES.shibbolethLoginForm);
 
```

Comment:
We need to put this into the ticket as a config update 😬 For now I just flagged the ticket so we don't forget.